nymea-app/ui/mainviews/WallboxView.qml

```diff
@@ -90,7 +94,7 @@ MainViewBase {
     EmptyViewPlaceholder {
         anchors.centerIn: parent
         width: parent.width - app.margins * 2
-        visible: !engine.thingManager.fetchingData && wallboxDevices.count == 0
+        visible: engine.thingManager.fetchingData && wallboxDevices.count == 0
```

Comment:
this change seems wrong to me, no?

Reply:
EmptyViewPlaceholder for inviting user to add a new wallbox device should be displayed when the number of available wallbox devices is zero.

Reply:
yes, I mean the fetchingData...

should probably be 
```
visible: !engine.thingManager.fetchingData && wallboxDevices.count == 0
```

Reply:
OK, will change that.